--- linux_net.py	2015-05-29 21:24:14.897241069 +0800
+++ linux_net.py.new	2015-05-29 21:24:08.554240809 +0800
@@ -1608,6 +1608,7 @@
             # bridge, which will either be the vlan interface, or a
             # physical NIC.
             _execute('ip', 'link', 'set', bridge, 'up', run_as_root=True)
+            _execute('ip', 'link', 'set', bridge, 'promisc', 'on', run_as_root=True)
 
         if interface:
             LOG.debug('Adding interface %(interface)s to bridge %(bridge)s',
